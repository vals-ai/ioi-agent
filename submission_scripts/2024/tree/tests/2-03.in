version https://git-lfs.github.com/spec/v1
oid sha256:f49a68d2378eeeec97095342d2466044b16eb3506c112be442d59965cf043532
size 22811
