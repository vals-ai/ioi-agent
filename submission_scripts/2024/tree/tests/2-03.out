version https://git-lfs.github.com/spec/v1
oid sha256:e0543c654c50406ad04109a18e1d871f3946a3c2b293e8c280fc14604eb39f6f
size 190
